sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -1166,6 +1173,36 @@ def unlifted_stages(stage):
       yield stage
 
 
+def _lowest_common_ancestor(a, b, context):
+  # type: (str, str, TransformContext) -> Optional[str]
+
+  '''Returns the name of the lowest common ancestor of the two named stages.
+
+  The provided context is used to compute ancestors of stages. Note that stages
+  are considered to be ancestors of themselves.
+  '''
+  assert a != b
+
+  parents = {
```

Comment:
Consider following for easier readability:
```
  parents = {
      child_id: parent_id
      for parent_id, parent in context.components.transforms.items()
      for child_id in parent.subtransforms
  }
```

Reply:
Yeah, this is an issue with yapf.